apps/prairielearn/elements/pl-file-upload/pl-file-upload.mustache

```diff
@@ -23,7 +23,7 @@ $(() => {
   {{/editable}}
   <div class="messages">
   </div>
-  <div class="file-upload-status" style="margin-top: 1ex;">
```

Comment:
1ex ~~ 1em ~~ mt-3 according to https://getbootstrap.com/docs/4.0/utilities/spacing/